src/cr/cube/cube.py

```diff
@@ -14,7 +19,9 @@
 
 from cr.cube.cubepart import CubePartition
 from cr.cube.dimension import AllDimensions
-from cr.cube.enums import CUBE_MEASURE, DIMENSION_TYPE as DT, NUMERIC_MEASURES
+from cr.cube.enums import CUBE_MEASURE
+from cr.cube.enums import DIMENSION_TYPE as DT
+from cr.cube.enums import NUMERIC_MEASURES
```

Comment:
This one was also fine as it was. Repeating the `from x.y.module import ...` prefix obscures the boundaries between source modules in the import list. Adding `--multi-line 8` will change it to the format we already use.